core/controllers/base.py

```diff
@@ -504,7 +522,14 @@ def init_csrf_secret(cls):
 
     @classmethod
     def _create_token(cls, user_id, issued_on):
-        """Creates a digest (string representation) of a token."""
+        """Creates a digest (string representation) of a token.
+        Args:
+          user_id: User Id.
```

Comment:
Indentation is wrong. Missing type information.

    user_id: str. The user_id for whom to create the token.